rl4co/models/zoo/matnet/encoder.py

```diff
@@ -51,7 +54,7 @@ def __init__(
         self.mix_W2 = nn.Parameter(mix_W2)
         self.mix_b2 = nn.Parameter(mix_b2)
 
-        self.out_proj = nn.Linear(embedding_dim, embedding_dim, bias=bias)
+        self.out_proj = nn.Linear(embedding_dim, embedding_dim, bias=True)
```

Comment:
Is there a reason why the bias is set to True? Because of the original code?

Reply:
oh yes, after struggling to obtain the results reported in the paper I tried to copy the architecture as much as possible, realizing that the bias in the projection of the heads does not have much effect. I would therefore revert this change.

Reply:
As you wish, this is minor!